app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t(".about")) %>
+<h1> <%= t(".about") %></h1>
+<p>
+  <%= t(".body_text",
+    link_home: link_to(t(".home_href"),"https://www.railstutorial.org/"),
+    link_book: link_to(t(".book_href"),"https://www.railstutorial.org/book"),
+    link_screencast: link_to(t(".screencast_href"),"http://screencasts.railstutorial.org/"),
+    link_ruby: link_to(t(".ruby_href"),"http://rubyonrails.org/")).html_safe %>
```

Comment:
`html_safe` code này dùng để làm gì, giải thích chị nghe xem

Reply:
hmtl_safe dùng để để chuyển các object thành string các chuỗi html. Không có html_safe nó sẽ in ra thẻ html chứ không trỏ đến địa chỉ.